docs/book/v6/tutorials/create-book-module.md

```diff
@@ -7,65 +7,102 @@ The below files structure is what we will have at the end of this tutorial and i
-* `src/Book/src/ConfigProvider.php` -  is a class that provides configuration for various aspects of the framework or application
-* `src/Book/src/RoutesDelegator.php` - a routes delegator is a delegator factory responsible for configuring routing middleware based on routing configuration provided by the application
-* `src/Book/src/InputFilter/BookInputFilter.php` - input filters and validators
-* `src/Book/src/InputFilter/Input/*` - input filters and validator configurations
+* `src/Book/src/Collection/BookCollection.php` – a collection refers to a container for a group of related objects, typically used to manage sets of related entities fetched from a database
+* `src/Book/src/ConfigProvider.php` – is a class that provides configuration for various aspects of the framework or application
+* `src/Book/src/Handler/GetBookCollectionHandler.php` – handler that reflects the GET action for the BookCollection class
+* `src/Book/src/Handler/GetBookResourceHandler.php` – handler that reflects the GET action for the Book entity
+* `src/Book/src/Handler/PostBookResourceHandler.php` – handler that reflects the POST action for the Book entity
+* `src/Book/src/InputFilter/CreateBookInputFilter.php` – input filters and validators
+* `src/Book/src/InputFilter/Input/*` – input filters and validator configurations
+* `src/Book/src/RoutesDelegator.php` – a routes delegator is a delegator factory responsible for configuring routing middleware based on routing configuration provided by the application
+* `src/Book/src/Service/BookService.php` – is a class or component responsible for performing a specific task or providing functionality to other parts of the application
+* `src/Core/src/Book/src/ConfigProvider.php` – is a class that provides configuration for Doctrine ORM
+* `src/Core/src/Book/src/Entity/Book.php` – an entity refers to a PHP class that represents a persistent object or data structure
+* `src/Core/src/Book/src/Repository/BookRepository.php` – a repository is a class responsible for querying and retrieving entities from the database
 
 ## Creating and configuring the module
 
 Firstly we will need the book module, so we will implement and create the basics for a module to be registered and functional.
 
-In `src` folder we will create the `Book` folder and in this we will create the `src` folder. So the final structure will be like this: `src/Book/src`.
+In `src` and `src/Core/src` folders we will create one `Book` folder and in those we will create the `src` folder. So the final structure will be like this: `src/Book/src` and `src/Core/src/Book/src`.
 
-In `src/Book/src` we will create 2 php files: `RoutesDelegator.php` and `ConfigProvider.php`. This files will be updated later with all needed configuration.
+In `src/Book/src` we will create 2 PHP files: `RoutesDelegator.php` and `ConfigProvider.php`. These files contain the necessary configurations.
 
 * `src/Book/src/RoutesDelegator.php`
```

Comment:
This section is duplicate - you are creating the `RoutesDelegator` towards the end of the tutorial.
Let's keep only that one!